server/container/cli-integration/src/test/java/org/apache/james/cli/QuotaCommandsIntegrationTest.java

```diff
@@ -61,39 +61,39 @@ public void tearDown() {
     }
```

Comment:
please rename this commit 14f04dfdbcf491c19e5d2cacd92787963c34708d (should be a fixup)

Reply:
will be done at squash time